mayastor/src/bdev/nexus/nexus_fn_table.rs

```diff
@@ -107,6 +107,10 @@ impl NexusFnTable {
                     trace!("{}: Dispatching RESET {:p}", nexus.bdev.name(), io);
                     nexus.reset(io, &ch)
                 }
+                io_type::FLUSH => {
+                    trace!("{}: Dispatching FLUSH {:p}", nexus.bdev.name(), io);
```

Comment:
Maybe you want to comment-out the trace? I left the reset un-commented because resets were rare enough, but read and write don't generate trace output, is it needed for flush? 
(BTW the commented-out trace statements are wrong nexus.name -> nexus.bdev.name, could you fix?)

Reply:
Done. https://github.com/openebs/Mayastor/pull/291/commits/2ffdf29d5786644710db887ecdac4c22e4d7b755